clusterDeployer.py

```diff
@@ -182,6 +182,9 @@ def teardown(self):
         print(lh.run("virsh net-start default"))
         print(lh.run("systemctl restart libvirtd"))
 
+      print(lh.run(f"virsh pool-destroy guest_images"))
+      print(lh.run(f"virsh pool-undefine guest_images"))
+
```

Comment:
I understand why you want this. The reason that we don't have this is that you might deploy multiple clusters on the same host and they share a pool. Therefore, I will need to figure out how create a per-cluster pool instead. Then this part can go in too, but not yet.

Reply:
the above is useful in combination with the --images-path argument, because the same cluster will end-up re-using the existing pool even when setting a different images-path. e.g. if the previous dir is in a too small partition, even the later execution with a different dir in a suitable partition will fail if the old pool is not deleted in advance.